### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by OlenaTaras**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,43 +1,51 @@
...
 //Завдання 3
 
 	var x = document.getElementById('first').value;
 	var y = document.getElementById('second').value;
 
+	var rezult;
 	function summatask2(x,y){
-		rezult = x-y>0 ? x-y : y-x, // if you use rezult without var, you assign it to global scope, this is bad practice
-		alert(rezult)
+		rezult = x-y>0 ? x-y : y-x,
```
Comment (AndriySikora) [inline script.js:45]: what about my comment? maybe you'll try var rezult

Comment (OlenaTaras) [reply script.js:45]: see -> the row 43

Comment (AndriySikora) [reply script.js:45]: Please check this - http://www.w3schools.com/js/js_scope.asp
try to understand how global variables work. In your case you get global variable. Please don't polute global object with your rezult.

File: task2.html [hunk 2]
```diff
@@ -5,46 +5,36 @@
 	<title>All tasks</title>
 </head>
 <body>
-<!--please split with number of task-->
+
```
Comment (AndriySikora) [inline task2.html:8]: what about split with number of task

**Pull request merged**